src/main.js

```diff
@@ -48,21 +88,33 @@ const main = function() {
   }
 
   brain.stats.addRoot();
-  Memory.myRooms = _(Game.rooms).filter((r) => r.execute()).map((r) => r.name).value();
-  try {
-    Memory.myRooms.forEach(visualizer.myRoomDatasDraw);
-  } catch (e) {
-    console.log('Visualizer Draw Exeception', e);
+  // room execution via sigmoid function + every 10 ticks execute all rooms
+  if (Game.time % 10 === 0) {
```

Comment:
Just as an idea. Randomizing the room order, to make sure every room get executed at some time

Reply:
I don't wanted that. But we could make it as config...

Reply:
Keep it as is, just an idea

Reply:
added config.main.randomExecution